src/adapters/mqtt/index.ts

```diff
@@ -42,22 +42,27 @@ class MqttAdapter extends Adapter {
   }
 
   private getSecurityReqs() {
-    const securityRequirements = (this.AsyncAPIServer.security() || []).map(
-      (sec) => {
-        const secName = Object.keys(sec.json())[0]
-        return this.parsedAsyncAPI.components().securityScheme(secName)
+
+    let userAndPasswordSecurityReq
+    let X509SecurityReq
+
+    const securityRequirements = this.AsyncAPIServer.security().map(e => e.map(e => e.scheme()))
+
+    securityRequirements.forEach(security => {
+      for (const sec of security) {
+        if(sec.type() === 'userPassword') {
```

Comment:
shouldn't we make this options case-insensitive?

Reply:
According to docs it should be `userPassword`.

<img width="690" alt="Screenshot 2023-10-13 at 3 06 12 PM" src="https://github.com/asyncapi/glee/assets/41781438/4347b721-af7e-4c90-9631-edea24433533">